c3d-apply/workflow/scripts/seg_to_fcsv.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import nibabel as nib
+import numpy as np
+
+
+def compute_centroid(seg, affine, fid):
```

Comment:
I haven't gotten a chance to look at Greydon's segmentation code yet, but worth noting that this will only work properly if every segmented fiducial has only one connected component.